temoa/_internal/table_writer.py

```
@@ -96,79 +142,58 @@ def write_results(
-        (a) appends data (so scenario needs to be cleared elsewhere
-        (b) requires an iteration number to separate results
-        (c) only writes to MM required tables (obj, emissions right now)
-        :param M: solved model
-        :param iteration: an iteration index for scenario labeling
-        :return:
-        """
         if not self.tech_sectors:
             self._set_tech_sectors()
         self.write_objective(model, iteration=iteration)
-        # analyze the emissions to get the costs and flows
         e_costs, e_flows = poll_emissions(model=model)
```

Ruff (0.14.8)

Unpacked variable `e_costs` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)